packages/framework/src/Model/PriceList/PriceListFacade.php

```diff
@@ -123,6 +123,17 @@ public function delete(int $priceListId): void
     {
         $priceList = $this->getById($priceListId);
 
+        $originalProductIds = array_map(
+            static fn (PriceListProductPrice $priceListProductPrice) => $priceListProductPrice->getProduct()->getId(),
+            $priceList->getPriceListProductPrices(),
+        );
```

Comment:
nitpick, the same array map is used in the edit function, I would say `PriceList::getProducts()` or `PriceList::getProductIds()` might be a handy function even for the future

Reply:
Nide! thanks d4c04d5843